docs/executor/kubernetes.rst

```diff
@@ -20,7 +20,7 @@ Kubernetes Executor
 
 The kubernetes executor is introduced in Apache Airflow 1.10.0. The Kubernetes executor will create a new pod for every task instance.
 
-Example helm charts are available at ``scripts/ci/kubernetes/app/{airflow,volumes,postgres}.yaml`` in the source distribution.
+Example helm charts are available at ``scripts/ci/kubernetes/app/{secrets,volumes,postgres}.yaml`` in the source distribution.
```

Comment:
Those are not helm charts :). And to be perfectly honest they are not good examples. They are mainly meant to be for CI tests

Reply:
Yeah, I'm not familiar with helm, just with kube yaml so it seemed odd to me.  I can gladly change the wording to say `kubernetes files` or just take down my PR

Reply:
Happy if you could update it :).

Reply:
I think we should add information that these are files that are not suitable for production use. Many people read the documentation and then ask questions about these files.

Reply:
Updated ✅